### ISSUE TITLE
HasMod does not recognise standard mods like 66 and 67

### ISSUE THREAD

**Issue by raelianer** [Micky5991/VehicleInfoLoader#4]
https://github.com/Micky5991/VehicleInfoLoader/blob/3007790a8ddd94154ad2b5959edc51808041a69a/Data/VehicleManifest.cs#L59

The problem seems to be that these aren't in the js-files, because (I guess) every car can be modded that way. I think they have to be put manually in there together with the available values somehow.

**Comment by Micky5991** [comment]
You have to keep in mind, that this library is aimed to given an easy way to *load* information from https://github.com/Micky5991/GT-MP-vehicleInfo, Maybe there are some information missing.

What kind of mods are in the types 66 and 67?

**Comment by raelianer** [comment]
Yeah sure, I just thought it would be nice to have all available information without additional coding. Seems to be mainly mods related to coloring:
Colour 1 - 66 (0 to 74)
Colour 2 - 67 (0 to 74)
Window Tint - 69
Dashboard Color - 74
Trim Color - 75
https://wiki.gt-mp.net/index.php?title=Vehicle_Mods

**Comment by Micky5991** [comment]
Okay, I see. I currently have a extended version ready to be downloaded, but the problem with this is, that the names are missing most of the times for new values.

If you could provide some data I could add it to all vehicles.
[vehicleInfo-de.zip](https://github.com/Micky5991/VehicleInfoLoader/files/1431029/vehicleInfo-de.zip)

**Comment by raelianer** [comment]
I don't know what to do exactly, but I could just go to GTA online and write german names of the values down, if that is what's needed.

**Comment by Micky5991** [comment]
No, I need the actual language-keys that are used to detemine the resulting texts in specific languages. For example the names for the window tints are listed like this:

```JSON
"69": {
	"amount": 5,
	"list": {
		"0": {
			"name": "CMOD_WIN_0",
			"localizedName": "Keine",
			"flags": ["stock"]
		},
		"1": {
			"name": "CMOD_WIN_4",
			"localizedName": "Geschwärzt",
			"flags": []
		},
		"2": {
			"name": "CMOD_WIN_3",
			"localizedName": "Limousine",
			"flags": []
		},
		"3": {
			"name": "CMOD_WIN_2",
			"localizedName": "Dunkles Rauchglas",
			"flags": []
		},
		"4": {
			"name": "CMOD_WIN_1",
			"localizedName": "Helles Rauchglas",
			"flags": []
		}
	}
}
```

The language-keys are `CMOD_WIN_0` - `CMOD_WIN_4`

**Comment by raelianer** [comment]
Okay, I don't know where to get these, I just started using GT-MP.

**Comment by Micky5991** [comment]
You could give me some already translated strings, maybe I can find those out while searching through the game-files.

**Comment by raelianer** [comment]
Sure, I'll do that.
